src/Hl7.Fhir.Support.Tests/Specification/MultiTerminologyServiceTests.cs

```diff
@@ -72,11 +72,63 @@ public async Task MultipleTerminologyServicesErrorHandlingTest()
             await call.Should().ThrowAsync<FhirOperationException>().Where(e => ((AggregateException)e.InnerException).InnerExceptions.Count == 2);
         }
 
-        private ITerminologyService setupMockTermService(Parameters output)
+        [TestMethod]
+        public async Task OrderableTerminologyServiceTest()
```

Comment:
This will test the happy flow. But what if 
- there are no TermServs registered at all?
- all the TermServs are not matching?